app/migrations/20250306131022-adds_project_id_to_city_table.cjs

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn("City", "project_id", {
```

Comment:
You also need to add a foreign key constraint here ideally. See the other migrations for the setup (you need to undo migrations to re-run this locally then probably).